### PR TITLE
SAGE-1415: log scoreboard on service start and on reboot/shutdown

### PR DESCRIPTION
**Pull request by jswantek**

### CODE REVIEW

File: ROOTFS/usr/bin/waggle_network_watchdog.py [hunk 1]
```diff
@@ -234,20 +252,20 @@ def hard_reboot_action():
...
+            shutdown_os_helper(nwwd_config)
         else:
             logging.warning("executing media switch recovery action")
 
-            write_current_resets(nwwd_config.hard_reset_file, 0)
-            write_current_resets(nwwd_config.soft_reset_file, 0)
-            write_current_resets(nwwd_config.network_reset_file, 0)
-
-            if int(nwwd_config.current_media) == 0:
+            if int(nwwd_config.current_media) == MEDIA_MMC:
```
Comment (seanshahkarami) [inline ROOTFS/usr/bin/waggle_network_watchdog.py:259]: I noticed this was moved up from where it was before.

Just curious if you'd actually seen a case where doing thing after actually led to a problem? (Ex. unable to write to disk)

**Review by seanshahkarami — APPROVED** [review]
Looks good. It'll be nice to have this if we have to debug later.

File: ROOTFS/usr/bin/waggle_network_watchdog.py (hunk 1 above)
Comment (jswantek) [reply ROOTFS/usr/bin/waggle_network_watchdog.py:259]: nope.  this was purely speculation and I can leave this change out.

Comment (seanshahkarami) [reply ROOTFS/usr/bin/waggle_network_watchdog.py:259]: It's fine - the tests are passing and I don't have any reason to believe either way was more "right", so definitely leave it. :)

**Pull request merged**